sdk/src/accounts/webSocketAccountSubscriberV2.ts

```diff
@@ -141,8 +155,8 @@ export class WebSocketAccountSubscriberV2<T> implements AccountSubscriber<T> {
 					abortSignal: abortController.signal,
 				});
 
-			// Start notification loop without awaiting
-			this.handleNotificationLoop(subscription);
+			// Start notification loop with the subscription promise
+			this.handleNotificationLoop(subscriptionPromise);
```

Comment:
feels weird to be accepting a promise for a function

maybe could use `.then` instead and pass the resolved Promise to `this.handleNotificationLoop`?

Reply:
ok it is totally weird but I did this for performance haha. I didn't want to wait for the WS subscription to succeed in the overall .subscribe() path because it slows down the app from rendering, while we can put this off to a separate thread because we do an initial fetch of the data anyway.